### ISSUE TITLE
sound: documentation for other teams to use sound in their module

### ISSUE THREAD

**Issue by parkerbroz** [uchicago-cs/chiventure#1436]
In this issue we aim to create documentation for other teams to use in their sound module. We will update the README document for this purpose. We are planning on collaborating with the RPG battle team on sound integration (for reference).

assignees: Jake, Rhea, Parker

**Comment by jwschroeder21** [comment]
Victor reached out to the RPG team, and they requested documentation so that they could use the functions in sound.h/sound.c

**Comment by jwschroeder21** [comment]
Update: added changes to the read me to specify how other teams can use the sound modules. I mainly discussed the use of mp3 files, wav files, sound_t declaration, and what to do if a lot of sounds are required. Special thanks to Victor for brainstorming with me. Waiting on PR approval before closing the issue.

Also removed Victor as an assignee because he will do the PR review.

**Comment by jwschroeder21** [comment]
Closing statement: we created documentation for other teams to use in their sound module. the documentation discussed mp3 files, wav files, sound_t declaration, and Mix_Chunk + Mix_Music specification. we will meet with the reg battle team wednesday at 5pm to further discuss this in person, but hopefully the documentation should help!

Current status: Waiting on Elizabeth's approval of the PR, and then will close the issue!
